cypress/integration/tests/migration/applicationinventory/analysis/binary_analysis.test.ts

```diff
@@ -45,18 +45,18 @@ describe("Binary Analysis", { tags: "@tier1" }, () => {
 
         // Perform login
         login();
-        deleteApplicationTableRows();
-
-        //Disable all proxy settings
-        Proxy.disableAllProxies();
-
-        // Enable HTML anc CSV report downloading
-        let generalConfig = GeneralConfig.getInstance();
-        generalConfig.enableDownloadHtml();
-        generalConfig.enableDownloadCsv();
+        // deleteApplicationTableRows();
+        //
+        // //Disable all proxy settings
```

Comment:
Why are we commenting this ?

Reply:
Sorry by mistake was commented. Thanks for pointing out!